google/resumable_media/requests/download.py

```diff
@@ -19,6 +19,9 @@
 from google.resumable_media.requests import _helpers
 
 
+_SINGLE_GET_CHUNK_SIZE = 8192
```

Comment:
@thorbla, if you do make it here, some context:

This change allows use of [`iter_content()`][1] with `requests` to avoid having to read the entire file into RAM before writing it to disk.

[1]: http://docs.python-requests.org/en/master/user/advanced/#body-content-workflow